src/shared/components/matrix/myProfileForm/myProfileForm.component.tsx

```diff
@@ -22,6 +26,14 @@ export const MyProfileForm = () => {
     submit,
   } = useMyProfileForm();
 
+  useEffect(() => {
```

Comment:
All of the logic for MyProfileForm is inside the useMyProfileForm hook. Can you move this logic there? We will then have a clear separation.I also believe that we can handle toast logic it in the submit function, so that we don't need to create these 2 useEffects